src/uct/cuda/cuda_ipc/cuda_ipc_iface.c

```diff
@@ -80,6 +81,37 @@ static int uct_cuda_ipc_iface_is_reachable(const uct_iface_h tl_iface,
             *((const uint64_t *)dev_addr)) && ((getpid() != *(pid_t *)iface_addr)));
 }
 
+static double uct_cuda_ipc_iface_get_bw()
+{
+    CUdevice cu_device;
+    int major_version;
+    ucs_status_t status;
+
+    status = UCT_CUDADRV_FUNC(cuDeviceGet(&cu_device, 0));
+    if (status != UCS_OK) {
+        goto err;
+    }
+
+    status = UCT_CUDADRV_FUNC(cuDeviceGetAttribute(&major_version,
+                CU_DEVICE_ATTRIBUTE_COMPUTE_CAPABILITY_MAJOR, cu_device));
+    if (status != UCS_OK) {
+        goto err;
+    }
+
+    switch(major_version) {
+        case 6 /* Pascal */:
```

Comment:
please align `case` with `switch`, like
```
switch () {
case:
    foo();
```
probably we would need to include that recommendation to coding GD, because the alignment above is used thru the code

Reply:
also, maybe add constants for 6 and 7